data/quarg/quarg missions.txt

```diff
@@ -113,3 +113,134 @@ mission "First Contact: Kuwaru Efreti"
 			label destroy
 			`	It clicks its teeth together, which might be the Quarg equivalent of laughter. "It was hoped that one faction of robots would destroy the other, and leave for us less work to do, but they are sadly too evenly matched. Instead we merely restrain them from expanding, as one isolates a fire one cannot extinguish and for it to burn out waits." You thank it for its time, and the crowd of Quarg that has been gawking at you slowly disperses.`
 				decline
+
+
+
+mission "Quarg Satellites"
+	minor
+	name "Deploy Quarg Scanners"
+	description "Head to each of the marked systems to drop off the Quarg scanning satellites, then head to <destination> to collect your payment."
```

Comment:
maybe change 
>"drop off"

into "deploy" , considering this is more of a recon mission than a cargo mission (as seen later on)